Automattic/wp-calypso
# Add/all jetpack product benefits in checkout

Adds a product features list to checkout for all Jetpack products when a single product is in the cart. No list will show if there are more than one item in the cart

NOTE: These lists were pulled from the pricing page and edited slightly to fit the context. For the Security bundles, a few items were removed to make the list shorter and make sense, and for Complete, only the list of products is shown. For more context, see this post: pbNhbs-5vs-p2

Related to #

## Proposed Changes

* Add more product lists for the rest of the Jetpack products that will be shown in the order summary in checkout
* Update tests to be more dynamic

## Testing Instructions

1. Check out these changes
2. Start your local environment with `yarn start`
3. Go to http://calypso.localhost:3000/checkout/jetpack/jetpack_boost_monthly and ensure the product feature list shows up
4. Go to http://calypso.localhost:3000/checkout/jetpack/jetpack_backup_t2_yearly and ensure the product feature list does NOT show up
5. Try out some other products with the URL format of http://calypso.localhost:3000/checkout/jetpack/{product_slug}. I'd recommending also checking out `jetpack_complete` and `jetpack_security_t1_yearly` (or monthly, idc). See the reference below for the product slugs
6. Make sure new tests are passing, and that they make sense as far as what to test

NOTE: No feature list should show if multiple products are in the cart

Product slugs that should show a feature list in the order summary:

* jetpack_boost_yearly
* jetpack_boost_monthly
* jetpack_backup_t1_yearly
* jetpack_backup_t1_monthly
* jetpack_scan
* jetpack_scan_monthly
* jetpack_anti_spam
* jetpack_anti_spam_monthly
* jetpack_search
* jetpack_search_monthly
* jetpack_videopress
* jetpack_videopress_monthly
* jetpack_social_basic_yearly
* jetpack_social_basic_monthly
* jetpack_social_advanced_yearly
* jetpack_social_advanced_monthly
* jetpack_security_t1_yearly
* jetpack_security_t1_monthly
* jetpack_complete
* jetpack_complete_monthly

Product slugs that should NOT show a feature list in the order summary

* jetpack_backup_t2_yearly
* jetpack_backup_t2_monthly
* jetpack_security_t2_yearly
* jetpack_security_t2_monthly
* Anything else

## Pre-merge Checklist
